docs/user/openstack/install_upi.md

```diff
@@ -156,25 +148,42 @@ python3 -m pip install --upgrade pip
 python3 -m pip install yq openstackclient openstacksdk netaddr
 ```
 
+Make sure, that none of copied playbooks contains an alias for ansible
+utility filters:
```

Comment:
We need to replace `ansible.utils.` prefix for filters on all playbooks, as RHEL8 still have Ansible 2.9, so that all the filters are already there - no need for installing ansible.utils collection from ansible galaxy.